docs/docs/guide/2_api-reference/data/get.md

```diff
@@ -11,6 +11,6 @@ data.get('sw-product-detail__product').then((data) => {
 ```
 
 #### Parameters
-| Name                 | Required | Description                                                                                           |
-| :------------------- | :------- | :---------------------------------------------------------------------------------------------------- |
-| `id`                 | true     | The unique id of the dataset you want to receive                                                      |
+| Name | Required | Description                                      |
```

Comment:
Unneeded changes

Reply:
Oh, derp. PHPStorm autofixed it and I slept :D